src/ralph/admin/filters.py

```diff
@@ -414,6 +414,29 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class IPFilter(SimpleListFilter):
```

Comment:
could you make a diff of your branch and current `feature/deployment`? many of those changes were already merged